data/lib/core/constants.lua

```diff
@@ -15,6 +15,15 @@ VOCATION_ELDER_DRUID = 6
 VOCATION_ROYAL_PALADIN = 7
 VOCATION_ELITE_KNIGHT = 8
 
+CYCLOPEDIA_SKILL_MAGIC = 1
+CYCLOPEDIA_SKILL_SHIELDING = 6
+CYCLOPEDIA_SKILL_DISTANCE = 7
+CYCLOPEDIA_SKILL_SWORD = 8
+CYCLOPEDIA_SKILL_CLUB = 9
+CYCLOPEDIA_SKILL_AXE = 10
+CYCLOPEDIA_SKILL_FIST = 11
+CYCLOPEDIA_SKILL_FISHING = 13
```

Comment:
Do we have any idea what is inbetween i.e. what is 2-5 and 12?

Reply:
i have no idea what they are, i just know that the current mapping works as it should, in fact all the code works as expected.